modules/trusted-firmware-m/tfm_boards/common/assert.c

```diff
@@ -24,7 +24,7 @@
 	"\nRefer to the documentation section 'TF-M partition alignment requirements'" \
 	"\nfor more information.\n\n"
 
-#error "TF-M non-secure start address is not aligned to SPU HW requirements"
+#warning"TF-M non-secure start address is not aligned to SPU HW requirements"
```

Comment:
this should be an error because it deals with hardware registers

Reply:
I tried to inform that this is a temporary work around until the PR which fixes this is merged.

> PR https://github.com/nrfconnect/sdk-nrf/pull/15210 needs to be merged before this and I'll rebase it on-top when that's done. 

This is only a temporary workaround just to have it compiling since the limitations doesn't apply for nRF54L15

Reply:
Rebased PR on-top now.